R/help/helpServer.R

```diff
@@ -1,11 +1,24 @@
 # get values from extension-set env values
 lim <- Sys.getenv("VSCR_LIM")
 
+SLEEP_DURATION <- 1
+
+NEW_PACKAGE_STRING <- "NEW_PACKAGES"
+
 cat(
     lim,
     tools::startDynamicHelp(),
     lim,
     sep = ""
 )
 
-while (TRUE) Sys.sleep(1)
+currentPackages <- .packages(all.available = TRUE)
+
+while (TRUE) {
+    Sys.sleep(1)
+    newPackages <- .packages(all.available = TRUE)
+    if (!identical(currentPackages, newPackages)) {
```

Comment:
It works well with changes in installed packages. Sometimes, the names of installed package do not change, but a new version is installed. I guess it might make some sense to use `newPackages <- installed.packages()` here so that if any bit of the information of all packages change, the aliases should be reloaded.

However, it could not handle installing the same version number from GitHub instead of CRAN.

Reply:
Looks like we could use `newPackages <- installed.packages(fields = "Packaged")` and compare `currentPackages` with `newPackages[, c("Version", "Packaged")]` should be enough to capture all these changes?

Reply:
Seems to be working nicely, thanks!